src/SimpleChocolateyServer/SimpleChocolateyServer.csproj

```diff
@@ -156,7 +157,6 @@
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
   </PropertyGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(VSToolsPath)\WebApplications\Microsoft.WebApplication.targets" Condition="'$(VSToolsPath)' != ''" />

```

Comment:
@ferventcoder the problem was here. When running build.bat, this resolved to a v11 folder, which I don't have on my VM as I only have VS 2017 installed.

Reply:
:+1: